lib/src/abstracts/media_picker_service.dart

```diff
@@ -4,10 +4,15 @@
 
 import 'dart:typed_data';
 
+import 'package:file_picker/file_picker.dart';
+
 abstract class MediaPickerService {
   /// Returns [Uint8List] based on given [ImageSource].
   Future<Uint8List?> pickImageFile();
 
   /// Returns [Uint8List] based on given [VideoSource].
   Future<Uint8List?> pickVideoFile();
+
+  /// Returns [Uint8List] based on given [File].
+  Future<FilePickerResult?> pickFile(List<String> fileExtensions);
```

Comment:
Uint8List plz

Reply:
See the comment to your other uint8list comment